artifact-management/commands/gem-test.yml

```diff
@@ -1,44 +1,98 @@
-description: Perform the actual gem build
+description: Perform gem test with system Ruby, or with one or more Ruby
+  versions (using RVM) if ruby_versions is set. ruby_versions should be a
+  comma-separated string with no spaces, such as "2.2.10,2.3.8,2.5.8"
 parameters:
-  bundler_version:
-    description: The Bundler version that should be used for "bundle install" and the actual test command
+  bundler_1_version:
+    description: The precise Bundler 1.x version
     type: string
     default: 1.17.3
-  gem_repo_host:
-    description: The host to which built packages should be pushed. 
-    type: env_var_name
-    default: AVVO_GEM_REPO_HOST
+  bundler_2_version:
+    description: The precise Bundler 2.x version
+    type: string
+    default: 2.1.4
   gem_repo_key:
     description: The :rubygems_api_key used to access the gem repository host
     type: env_var_name
     default: AVVO_GEM_REPO_TOKEN
   gem_test_dir:
-    description: The directory in which the gem tesets should run
+    description: The directory in which the gem tests should run
     type: string
     default: .
   packagecloud_read_token:
     description: The token to be used when fetching packages from Packagecloud.
     type: env_var_name
     default: PACKAGECLOUD_READ_TOKEN
-  test_command:
-    description: The command that should be used to run tests
+  ruby_versions:
+    description: The ruby versions to be used to test the gem.
```

Comment:
Add to this description: "If set, RVM is assumed to be installed and the install/test commands will be prefaced with `rvm <<ruby_versions> do ...`. If unset (the default), the system ruby will be called directly."